src/MarginTrading.AssetService.Services/ProductsService.cs

```diff
@@ -185,7 +189,7 @@ public async Task<Result<ProductsErrorCodes>> UpdateBatchAsync(List<Product> pro
 
         public async Task<Result<ProductsErrorCodes>> DeleteBatchAsync(List<string> productIds, string username)
         {
-            var existing = await _repository.GetAllAsync(null, productIds.ToArray());
+            var existing = await _repository.GetAllAsync(null, null, productIds.ToArray());
```

Comment:
Same, makes sense to use named arguments for better readability.

```suggestion
            var existing = await _repository.GetAllAsync(mdsCodeFilter: null, mdsCodes: null, productIds.ToArray());
```

Reply:
done